app/src/helpers/windowHelpers.test.ts

```diff
@@ -97,6 +97,8 @@ describe('createNewTab', () => {
 });
 
 describe('injectCSS', () => {
+  jest.setTimeout(10000);
+
```

Comment:
Was this added during debugging and forgotten, or is it actually necessary?

Reply:
It seemed necessary on my Windows machine, though it may just be slow. 😆